src/components/dashboard/Claim/PhaseOne.js

```diff
@@ -4,80 +4,97 @@ import { View } from 'react-native'
 import numeral from 'numeral'
 
 import Section from '../../common/layout/Section'
-import Text from '../../common/view/Text'
 import BigGoodDollar from '../../common/view/BigGoodDollar'
 
 import { weiToGd } from '../../../lib/wallet/utils'
 import { openLink } from '../../../lib/utils/linking'
+import { isSmallDevice } from '../../../lib/utils/mobileSizeDetect'
 import ButtonBlock from './ButtonBlock'
 
 // eslint-disable-next-line require-await
 const openLearnMoreLink = async () => openLink('https://w3.gooddollar.org/learn/ubi')
 
-const ClaimPhaseZero = ({
+const calcFontSize = fontSize => {
+  if (isSmallDevice) {
+    return fontSize / 1.3
+  }
+
+  return fontSize
+}
+
+const ClaimPhaseOne = ({
   handleClaim,
   faceRecognition,
   styles,
   isCitizen,
   entitlement,
   nextClaim,
+  theme,
   claimedToday: { amount, people },
 }) => (
-  <>
-    <Section.Stack style={styles.mainText}>
-      <View style={styles.mainTextBorder}>
-        <Section.Text>
-          <Section.Text color="surface" fontFamily="slab" fontWeight="bold" fontSize={40}>
-            {entitlement ? `Claim Your\nDaily Share` : `Just a Few More\nHours To Go...`}
-          </Section.Text>
-        </Section.Text>
-      </View>
+  <Section.Stack style={styles.mainContainer}>
+    <View style={styles.headerContentContainer}>
+      <Section.Text
+        color="surface"
+        fontFamily="slab"
+        fontWeight="bold"
+        fontSize={calcFontSize(40)}
```

Comment:
could we do this in withStyles ?
also why we have to calculate ? could we just to define 2-3 different styles for different sizes ?

Reply:
yes we can